operators/src/pro/cache/tile_cache.rs

```diff
@@ -148,13 +517,16 @@ pub struct CacheEntry {
 
 identifier!(CacheEntryId);
 
+impl ByteSize for CacheEntryId {}
+
 impl CacheEntry {
     /// Return true if the query can be answered in full by this cache entry
     /// For this, the bbox and time has to be fully contained, and the spatial resolution has to match
     pub fn matches(&self, query: &RasterQueryRectangle) -> bool {
         (self.query.spatial_bounds == query.spatial_bounds
             || self.query.spatial_bounds.contains(&query.spatial_bounds))
-            && self.query.time_interval.contains(&query.time_interval)
+            && (self.query.time_interval == query.time_interval
```

Comment:
the contains method already checks for equality or am I wrong?

Reply:
i guess thats an open discussion at the moment?

Reply:
a0fb56f2e65b6ef68a5b30f60f1e6e459fc04b97 now it does